drivers/postgres_example_test.go

```diff
@@ -0,0 +1,114 @@
+package drivers
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+)
+
+const CreateTable = `CREATE TABLE IF NOT EXISTS pg_data_types (
```

Comment:
Can you add some of them as NOT NULL, just to handle that case as well in the tests.